packages/app/src/app.ts

```diff
@@ -152,7 +153,7 @@ export class App<Req extends Request = Request, Res extends Response = Response>
 
   /**
    * Render a template
-   * @param file What to render
+   * @param name What to render
```

Comment:
seems like unrelated change

Reply:
My IDE was giving me grief about the jsdoc parameter being missing.